src/test/operators/table_scan_string_test.cpp

```diff
@@ -0,0 +1,316 @@
+#include <iostream>
+#include <map>
+#include <memory>
+#include <set>
+#include <string>
+#include <utility>
+
+#include "../base_test.hpp"
+#include "gtest/gtest.h"
+
+#include "expression/evaluation/like_matcher.hpp"
+#include "operators/abstract_read_only_operator.hpp"
+#include "operators/get_table.hpp"
+#include "operators/table_scan.hpp"
+#include "operators/table_scan/like_table_scan_impl.hpp"
+#include "storage/chunk_encoder.hpp"
+#include "storage/storage_manager.hpp"
+#include "storage/table.hpp"
+#include "types.hpp"
+
+using namespace std::string_literals;  // NOLINT
+
+namespace opossum {
+
+class OperatorsTableScanStringTest : public BaseTest, public ::testing::WithParamInterface<EncodingType> {
```

Comment:
Is this the same tests as renamed in this PR: https://github.com/hyrise/hyrise/pull/1016/files ? Why is the entire file marked as added?

Reply:
Merge f... screw-up